usagov/vote-gov-drupal
# Revert "VOTE-744 & 745 mega menu configuration (#518)"

This reverts commit def4042d30a235e1d13db5016428caecd1995baf.



## Jira ticket

https://cm-jira.usa.gov/browse/VOTE-782

## Description

Revert megamenu config and remove menu_item_extras module

## Deployment and testing

### Pre-deploy

1. uninstall menu_items_extras module

### Post-deploy

1. run `lando retune`

### QA/Test

NOTE: you will need the latest DB.
1. visit http://vote-gov.lndo.site/ and test the dropdown menu for How to Vote main menu item. Make sure it shows the submenu items. 
2. Visit a Voter Guide page and confirm the menu shows in the sidebar.

## Checklist for the Developer




## Checklist for the Peer Reviewers
